common/src/test/java/com/microsoft/identity/common/internal/providers/microsoft/nativeauth/utils/MockApiResponseType.kt

```diff
@@ -0,0 +1,75 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+package com.microsoft.identity.common.internal.providers.microsoft.nativeauth.utils
+
+/**
+ * Enumerates all of the different types of responses received by Mock API for Native Auth
+ */
+enum class MockApiResponseType(val stringValue: String) {
```

Comment:
Is this a field inside the response?

Reply:
Is this a header or something in the body?

Reply:
What do we mean by response type though? Is that a field somewhere in the HTTP response?

Reply:
Response type would mean the signature of the response. For example `ATTRIBUTES_REQUIRED` response type would include the list of attributes that should be supplied by the client.